ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -276,3 +276,7 @@ def greater_than(self, y, name=None):
     @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
     def rsqrt(self, name=None):
         return ivy.reciprocal(ivy.sqrt(self._ivy_array))
+
+    @with_unsupported_dtypes({"2.5.0 and below": ("float16", "bfloat16")}, "paddle")

```

Comment:
This is failing for "int16" as it is not supported by paddle, you will probably come across more unsupported dtypes failures after adding this

Reply:
![image](https://github.com/unifyai/ivy/assets/83540902/be59d0a8-2032-451f-81bc-1f9760f07473)

Hey @zaeemansari70 i am getting this error while testing the method. i am using pytest in vscode (wsl2-ubuntu-20.04). is there any different command to run test in cli.

thank you